content-service/src/main/java/io/meeds/news/service/impl/NewsServiceImpl.java

```diff
@@ -394,7 +390,20 @@ public void publishNews(News newsToPublish, String publisher) throws Exception {
    */
   @Override
   public void unpublishNews(String newsId, String publisher) throws Exception {
+    News news = getNewsArticleById(newsId);
+    newsTargetingService.deleteNewsTargets(news, publisher);
 
+    NewsPageObject newsPageObject = new NewsPageObject(NEWS_METADATA_PAGE_OBJECT_TYPE, news.getId(), null);
+    MetadataItem newsMetadataItem = metadataService.getMetadataItemsByMetadataAndObject(NEWS_METADATA_KEY, newsPageObject).stream().findFirst().orElse(null);
+
+    if (newsMetadataItem != null) {
+      Map<String, String> properties = newsMetadataItem.getProperties();
+      if (properties != null) {
+        properties.put(PUBLISHED, String.valueOf(false));
+        properties.remove(NEWS_PUBLISH_DATE);
```

Comment:
can we check if we need this property publishDate ?
can we check also if property publicationState is needed or should be replaced by another check ?

Reply:
This property needs to be displayed on the news list view item !